source/manual/update_popular_links.html.md

```diff
@@ -6,12 +6,16 @@ layout: manual_layout
 parent: "/manual.html"
 ---
 
-Popular links on GOV.UK are shown on the home page.
+Popular links on GOV.UK as shown on the home page under 'Popular on GOV.UK'.
 
 ![Image of the popular links on the GOV.UK homepage](images/popular-links-on-homepage.jpeg)
 
-## Updating the popular links on the homepage
+## How to access and edit popular links
 
-Popular links can be found in [config/locales/en.yml on frontend](https://github.com/alphagov/frontend/blob/80ec5cb2840086e7f073ceeba89d5c07d581a02d/config/locales/en.yml#L363-L373)
+Popular links can be accessed and edited via the 'Homepage' link on Signon, for this we need the 'homepage_editor' permission inside the Mainstream Publisher app.
 
-[Example PR](https://github.com/alphagov/frontend/pull/3155)
+### Note
+
+There is no separate app for homepage, all the code for homepage lives inside Mainstream Publisher app and hence any user permission required for homepage is also managed via Mainstream Publisher app.
```

Comment:
I'd probably either refer to Mainstream Publisher as "Mainstream Publisher" or "the Mainstream Publisher app" (so either add "the" to the beginning, or remove "app".